deployment/united-manufacturing-hub/values.yaml

```diff
@@ -961,6 +961,55 @@ _000_commonConfig:
   kafkaStateDetector:
     enabled: false
 
+_001_customMicroservices:
+  -
+      name: example
+      image: hello-world:latest
+
+      # imagePullPolicy: Always is recommended when using latest tags. Otherwise, please use IfNotPresent
+      imagePullPolicy: Always
+
+      # these will be passed as environment variables
+      env:
+        - name: LOGGING_LEVEL
+          value: PRODUCTION
+
+      # this will expose port 80 on the host on port 8080
+      port: 80
+      externalPort: 8080
+
+      # startupProbe, livenessProbe, check whether port 8080 "/health" is accessible.
+      # If not (e.g., due to container stuck), the container will be restarted
+      # startupProbe: 100
+      # livenessProbe: 50
+
+      # statefulEnabled will automatically mount a volume under /data, which can be used to store data.
+      # Recommendation is to turn it off
+      statefulEnabled: true
+  -
+      name: example2
+      image: hello-world:latest
+
+      # imagePullPolicy: Always is recommended when using latest tags. Otherwise, please use IfNotPresent
+      imagePullPolicy: Always
+
+      # these will be passed as environment variables
+      env:
+        - name: LOGGING_LEVEL
+          value: PRODUCTION
+
+      # this will expose port 80 on the host on port 8080
+      port: 81
+      externalPort: 8081
+
+      # startupProbe, livenessProbe, check whether port 8080 "/health" is accessible.
```

Comment:
In the current implementation, it checks for the variable 'port'. Lets set that to port 8080 or probePort

Reply:
49ff0dc53a18051400e66102ca7e9ee6e0647e56